lemur/certificates/schemas.py

```diff
@@ -359,6 +362,22 @@ def handle_subject_details(self, data):
             if field in data and data[field] is None:
                 data.pop(field)
 
+    @post_dump
+    def handle_certificate(self, cert):
+        # Plugins may need to modify the cert object before returning it to the user
+        if cert['authority'] is None:
+            if cert['root_authority'] is None:
+                plugin = None
+            else:
+                # this certificate is an authority
+                plugin = plugins.get(cert['root_authority']['plugin']['slug'])
+        else:
+            plugin = plugins.get(cert['authority']['plugin']['slug'])
+        if plugin:
+            plugin.wrap_certificate(cert)
+        if 'root_authority' in cert:
```

Comment:
@GnunuX quick clarification question on this workaround: 

is the reason for deleting `root_authority` that it has already served its purpose?
```
                # this certificate is an authority
                plugin = plugins.get(cert['root_authority']['plugin']['slug'])
```

what would be side effect of not deleting it?

Reply:
I don't see any side effect.
I'll suppress this.

Reply:
ok, sounds good.